Backend development

BillieVijo opened the pull request.

BillieVijo committed:
modifying frontends

BillieVijo committed:
category making feature

include controller migration and views for categories

BillieVijo committed:
Finishing Categories

include  adding, category, modifying it and deleting

BillieVijo committed:
User Controller

view controller routes and migration

Jbac76 commented on app/Http/Controllers/CategoryController.php:
```diff
@@ -0,0 +1,103 @@
+
+use Illuminate\Http\Request;
+use Redirect;
+use DB;
```
Use a model

Jbac76 commented on app/Http/Controllers/CategoryController.php:
```diff
@@ -0,0 +1,103 @@
+     */
+    public function index()
+    {
+        $categories = DB::table('categories')->get();
```
Replace the DB:table() with a model

Jbac76 commented on app/Http/Controllers/CategoryController.php:
```diff
@@ -0,0 +1,103 @@
+        $data['description'] = $request->cat_desc;
+        $data['status'] = $request->status;
+
+        DB::table('categories')->insert($data);
```
Easier done with model

Jbac76 commented on app/Http/Controllers/CategoryController.php:
```diff
@@ -0,0 +1,103 @@
+     */
+    public function edit($id)
+    {
+        $cats = DB::table('categories')->where('id',$id)->first();
```
Again easier done with model, for later modifications purposes

Jbac76 commented on app/Http/Controllers/CategoryController.php:
```diff
@@ -0,0 +1,103 @@
+        $data['description'] = $request->cat_desc;
+        $data['status'] = $request->status;
+
+        DB::table('categories')->where('id',$id)->update($data);
```
Again use model

Jbac76 commented on database/migrations/2020_07_11_135906_create_admins_table.php:
```diff
@@ -15,6 +15,11 @@ public function up()
     {
         Schema::create('admins', function (Blueprint $table) {
             $table->id();
```
Avoid using create all together, am not a fan

Jbac76 committed:
Modifies category controller

Adds Models and updates the contoller

Jbac76 committed:
Fixes typo in category edit page

BillieVijo committed:
creates prouct mechanism

BillieVijo committed:
imports intervention for image uploads

import intervention and test so that it can be able to send images
peoduct image with thumbnail

BillieVijo committed:
Finishes delete products image

BillieVijo committed:
Creates HomeController for client

also creates its roote

BillieVijo committed:
Fixes Image visibility in welcome view

the fix touches the productcontroller, home controller,
